src/components/ConfigurationGame/index.js

```diff
@@ -46,11 +46,12 @@ class ConfigurationGame extends Component {
                     <p>Configuracion de la partida</p>
                     <div className="addPLayer">
                         <p>-Añade el numero de jugadores! (hasta 5 jugadores)</p>
-                        <Modal show={this.state.show} handleClose={this.hideModal} handleClick={() => this.addPlayer()}>
+                        <Modal show={this.state.show} handleClose={this.hideModal} handleClick={() => this.addPlayer(this.state.color[this.state.listPlayers.length], this.state.search)}>
                             <form>
                                 <label>
                                     Name:
-                                    <input type="text" value={this.state.search} onChange={this.updateSearch} name="name" />
+                                    <input type="text" value={this.state.search} onChange={this.updateSearch} name="name" color="color" />
```

Comment:
Creo que has confundido esto con una prop de un componente, o para qué lo añadiste?
```suggestion
                                    <input type="text" value={this.state.search} onChange={this.updateSearch} name="name" />
```